### DISCUSSION TITLE
Playing recorded .mbe files?

### DISCUSSION THREAD

**Discussion by vcrtech** [DSheirer/sdrtrunk#1541] [Q&A]
Is there a way to re-play recorded .mbe files?

**Comment by Allen-colton** [comment]
Yea just open the file location and click on the time and date u want

**Comment by Allen-colton** [comment]
Mine is latest verson and its been giving a c++ asention error

**Comment by vcrtech** [comment]
I open it with SDRTrunk or is there another included utility? My desired use case: I’d like to make/modify an external utility to play the .mbe’s in sequential order (similar to how the calls platform does it) rather than two channels at once, but I wasn’t sure if there was code that already does it. If I can play them sequentially based on file system time stamps, no audio is missed, but played in the order it’s received (privately as I have no interest in streaming it).Thanks!On May 8, 2023, at 03:21, Allen-colton ***@***.***> wrote:﻿
Mine is latest verson and its been giving a c++ asention error
